composer.json

```diff
@@ -30,23 +30,24 @@
     "require": {
         "php": "^7.4 || >=8.1",
         "composer-runtime-api": "^2.2",
-        "dealerdirect/phpcodesniffer-composer-installer": "^0.5 || ^0.7",
-        "phpstan/phpstan": "^1.9",
+        "dealerdirect/phpcodesniffer-composer-installer": "^0.5 || ^0.7 || ^1.0",
+        "phpstan/phpstan": "^1.9 || ^1.10",
```

Comment:
I did not want to force `^1.10` initially but I did want to allow it. Depending on how long this PR will be open, I might just enforce the `^1.10` since it will be stable enough by then.